develop/toolkit/parachains/spawn-chains/zombienet/get-started.md

```diff
@@ -840,6 +825,15 @@ You can use the `hrmp_channels` keyword to define further parameters for the XCM
         - `max_capacity` ++"number"++ - maximum capacity of the HRMP channel
         - `max_message_size` ++"number"++ - maximum message size allowed in the HRMP channel
 
+## Additional Support Resources
```

Comment:
Wdyt about moving these links to be cards in the Where to Go Next section? We just added an "external" class for linking to external resources. There's an example [here](https://github.com/polkadot-developers/polkadot-docs/blob/bf5852980c4cfb0f2fb095efe6a73f0e83cd6fe9/develop/smart-contracts/evm-toolkit/dev-environments/remix.md?plain=1#L122-L128)